scripts/regenerate-sbom.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
Could we add a few comments explaining what's going on here? The `jq` and `sed` commands in particular are difficult to parse unless you already know what the output looks like.

Reply:
Added.